.yamato/package-test.yml

```diff
@@ -0,0 +1,136 @@
+{% metadata_file .yamato/package.metafile -%}
+---
+
+{% for editor in test_editors %}
+{% for platform in test_platforms %}
+test_{{ platform.name }}_{{ editor.version }}:
+  name : Test {{ editor.version }} on {{ platform.name }}
+  agent:
+    type: {{ platform.type }}
+    image: {{ platform.image }}
+    flavor: {{ platform.flavor}}
+  variables:
+    UPMCI_PKG: "upm-ci-utils@stable"
+  commands:
+      - npm install {% if platform.name == "win" %}"%UPMCI_PKG%"{% else %}"$UPMCI_PKG"{% endif %} -g --registry {{ registry.npm }}
+      - upm-ci package test -u {{ editor.version }} --package-path package/com.unity.formats.usd --extra-create-project-arg=-upmNoDefaultPackages --extra-utr-arg="--api-profile=NET_4_6"
```

Comment:
Should this have --type package-tests?

Reply:
Good point, it looks like the [default is isolation tests](https://internaldocs.unity.com/package_development/upm-ci/commands/package-context/#test).
> Executes the complete suite of tests required for publishing (package validation suite, clean console, package has tests, and package tests)
These extra tests that will no longer be executed are part of vetting-tests, which runs during 'validation'. I think that's a better place for them to run, so I'll switch this to just package tests :)